packages/search/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
+ * Apache-2.0 */
+
+export {
+  search as agoSearch,
```

Comment:
if you're going to export it as `agoSearch`, why not just give the function that name to begin with?

Reply:
yeah good point